Makefile

```
@@ -153,12 +153,19 @@ multimod-verify: | $(MULTIMOD)
 	@echo "Validating versions.yaml"
 	$(MULTIMOD) verify
 
-COREPATH ?= "../opentelemetry-collector-contrib"
-OPTIONAL_CORE_PATH ?="../opentelemetry-collector"
+COREPATH ?= "../opentelemetry-collector"
+
 .PHONY: multimod-sync-core
 multimod-sync-core: multimod-verify
-	@[ ! -d $COREPATH ] || ( echo ">> Path to core repository must be set in COREPATH and must exist"; exit 1 )
-	$(MULTIMOD) sync -a -o ${COREPATH} -o ${OPTIONAL_CORE_PATH}
+	@[ ! -d COREPATH ] || ( echo ">> Path to core repository must be set in COREPATH and must exist"; exit 1 )
+	$(MULTIMOD) sync -a -o ${COREPATH}
+
+CORECONTRIBPATH ?= "../opentelemetry-collector-contrib"
```

nitpick
```suggestion
CONTRIBPATH ?= "../opentelemetry-collector-contrib"
```

Another nitpick: Remove the space below between the target and the variable.

---

Fixed nitpick. For the purpose of verifying all the upstream-dependencies are up to date by using ```multimode-sync```? Interesting